Scheduler/observing_blocks_maintel/AOS-OR4/BLOCK-277.json

```diff
@@ -0,0 +1,202 @@
+{
+    "name": "BLOCK-277",
+    "program": "BLOCK-277",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "run_command.py",
+            "standard": true,
+            "parameters": {
+                "component": "CCCamera",
+                "cmd": "play",
+                "parameters": {
+                    "playlist": "or4_day1_aos_sweep_cam_dx",
+                    "repeat": true
+                }
+            }
+        },
+        {
+            "name": "maintel/slew_to_azel.py",
+            "standard": true,
+            "parameters": {
+                "track_azel": {
+                    "az": 9.0,
+                    "el": 75.0
+                },
+                "rot_value": 0,
```

Comment:
I had the planning tool just continue tracking in between the dz sweep and this sweep, so maybe no reason to have a slew/track script again here?  We should probably adopt either:
- No slew/track script in any of the sweep blocks.  Do it separately before starting them.
- Add a slew/track to each block.
For the latter, note that the planning file has an updated az/el/rot since it's the result of tracking through the first sweep.  Here it should be: 
az 2.689826660387997 
alt 75.1490404707181
rot -5.648086981700885

https://github.com/jmeyers314/aos_block_planner/blob/main/OR4_AOS_schedule.ecsv#L163C101-L163C155

Reply:
I think it's cleaner to remove it. We will have to add the tracking script in the instructions on how to run the block